### DISCUSSION TITLE
What is your favorite drink? / 好きな飲み物を教えてください！

### DISCUSSION THREAD

**Discussion by ryosuke-sota** [NAIST-SE/AttractingContributors#1158] [2024 Attraction Contributions]
Please tell me.
Here↓
https://github.com/ryosuke-sota/AttractingContributors.git

**Comment by Ryota-keda** [comment]
I just made PR.
Please contribute to my project too.
https://github.com/Ryota-keda/Baseball

**Comment by kohanaist** [comment]
I have made a PR.
Could you contribute to my repo too?
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Reply by ryosuke-sota** [reply]
Thank you for your contribution, and I just made PR to your repo too (added as INFJ). 
Please take a look and confirm.
Thank you.

**Reply by anzhitingg** [reply]
Thank you very much！

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
